Failures in [node-test-pull-request/62962](https://ci.nodejs.org/job/node-test-pull-request/62962/) to [node-test-pull-request/63056](https://ci.nodejs.org/job/node-test-pull-request/63056/) that failed 2 or more PRs
(Generated with `ncu-ci walk pr --stats=true --markdown reports/2024-10-12.md`)

| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
| 2024-10-12 00:09 | 3       | 17      | 16       | 11      | 53      | 19.77%     |


### JSTest Failure

| Reason | <code>sequential/test-http-server-request-timeouts-mixed</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 7 ([https://github.com/nodejs/node/pull/54505/](https://ci.nodejs.org/job/node-test-pull-request/62964/), [https://github.com/nodejs/node/pull/55318/](https://ci.nodejs.org/job/node-test-pull-request/63000/), [https://github.com/nodejs/node/pull/55071/](https://ci.nodejs.org/job/node-test-pull-request/63002/), [https://github.com/nodejs/node/pull/55217/](https://ci.nodejs.org/job/node-test-pull-request/63015/), [https://github.com/nodejs/node/pull/55339/](https://ci.nodejs.org/job/node-test-pull-request/63016/), [https://github.com/nodejs/node/pull/55343/](https://ci.nodejs.org/job/node-test-pull-request/63045/), [https://github.com/nodejs/node/pull/55333/](https://ci.nodejs.org/job/node-test-pull-request/63052/)) |
| Appeared | [test-macstadium-macos11.0-arm64-3](https://ci.nodejs.org/job/node-test-commit-osx-arm/nodes=osx11/17103/console), [test-macstadium-macos11.0-arm64-4](https://ci.nodejs.org/job/node-test-commit-osx-arm/nodes=osx11/17076/console), [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/61492/console), [test-orka-macos11-x64-1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/61483/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62964/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63052/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx-arm/nodes=osx11/17103/console">Example</a></summary>

```
not ok 4206 sequential/test-http-server-request-timeouts-mixed
  ---
  duration_ms: 3173.83200
  severity: fail
  exitcode: 1
  stack: |-
    node:internal/assert/utils:281
        throw err;
        ^
    
    AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
    
      assert(request2.completed)
    
        at Timeout._onTimeout (/Users/iojs/build/workspace/node-test-commit-osx-arm/nodes/osx11/test/sequential/test-http-server-request-timeouts-mixed.js:108:5)
        at listOnTimeout (node:internal/timers:614:17)
        at process.processTimers (node:internal/timers:549:7) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: false,
      expected: true,
      operator: '=='
    }
    
    Node.js v23.0.0-pre
  ...

```
</details>

-------

| Reason | <code>node-api/test_threadsafe_function/test</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/55250/](https://ci.nodejs.org/job/node-test-pull-request/62971/), [https://github.com/nodejs/node/pull/55217/](https://ci.nodejs.org/job/node-test-pull-request/62972/), [https://github.com/nodejs/node/pull/55156/](https://ci.nodejs.org/job/node-test-pull-request/62981/), [https://github.com/nodejs/node/pull/55316/](https://ci.nodejs.org/job/node-test-pull-request/62998/), [https://github.com/nodejs/node/pull/55318/](https://ci.nodejs.org/job/node-test-pull-request/63000/)) |
| Appeared | [test-equinix-ubuntu2204_container-armv7l-2](https://ci.nodejs.org/job/node-test-binary-armv7l/RUN_SUBSET=native,nodes=ubuntu2204-armv7l/14086/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62971/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63000/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-armv7l/RUN_SUBSET=native,nodes=ubuntu2204-armv7l/14086/console">Example</a></summary>

```
not ok 144 node-api/test_threadsafe_function/test
  ---
  duration_ms: 360033.71500
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>wpt/test-compression</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/55171/](https://ci.nodejs.org/job/node-test-pull-request/62980/), [https://github.com/nodejs/node/pull/55316/](https://ci.nodejs.org/job/node-test-pull-request/62989/), [https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/63007/), [https://github.com/nodejs/node/pull/55239/](https://ci.nodejs.org/job/node-test-pull-request/63011/), [https://github.com/nodejs/node/pull/55332/](https://ci.nodejs.org/job/node-test-pull-request/63047/)) |
| Appeared | [test-ibm-ubuntu2204_sharedlibs_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46905/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46878/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46875/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46860/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62980/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63047/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46905/console">Example</a></summary>

```
not ok 4294 wpt/test-compression
  ---
  duration_ms: 1440114.90200
  severity: fail
  exitcode: -15
  stack: |-
    timeout
    [SKIPPED] compression-bad-chunks.tentative.any.js: Execution "hangs", ArrayBuffer and TypedArray is not accepted and throws, instead of rejects during writer.write
    [SKIPPED] decompression-bad-chunks.tentative.any.js: Execution "hangs", ArrayBuffer and TypedArray is not accepted and throws, instead of rejects during writer.write
    [SKIPPED] decompression-buffersource.tentative.any.js: ArrayBuffer and TypedArray is not accepted and throws, instead of rejects during writer.write
    [SKIPPED] idlharness-shadowrealm.window.js: ShadowRealm support is not enabled
    [SKIPPED] third_party/pako/pako_inflate.min.js: This is not a test file.
    [PASS] "a" should cause the constructor to throw
    [PASS] no input should cause the constructor to throw
    [PASS] non-string input should cause the constructor to throw
    [PASS] "a" should cause the constructor to throw
    [PASS] no inpu...
```
</details>

-------

| Reason | <code>parallel/test-net-write-fully-async-buffer</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/62976/), [https://github.com/nodejs/node/pull/55114/](https://ci.nodejs.org/job/node-test-pull-request/62984/), [https://github.com/nodejs/node/pull/55320/](https://ci.nodejs.org/job/node-test-pull-request/63040/), [https://github.com/nodejs/node/pull/55205/](https://ci.nodejs.org/job/node-test-pull-request/63054/)) |
| Appeared | [test-equinix-rhel8_container-arm64-1](https://ci.nodejs.org/job/node-test-commit-arm/nodes=rhel8-arm64/55312/console), [test-equinix_mnx-smartos20-x64-3](https://ci.nodejs.org/job/node-test-commit-smartos/nodes=smartos20-64/57278/console), [test-rackspace-win2022_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2022-COMPILED_BY-vs2022/30734/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62976/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63054/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-arm/nodes=rhel8-arm64/55312/console">Example</a></summary>

```
not ok 4113 parallel/test-net-write-fully-async-buffer
  ---
  duration_ms: 360046.43100
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-stream-readable-unpipe-resume</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/55241/](https://ci.nodejs.org/job/node-test-pull-request/62965/), [https://github.com/nodejs/node/pull/55114/](https://ci.nodejs.org/job/node-test-pull-request/62985/), [https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/62987/)) |
| Appeared | [test-equinix-ubuntu2204_container-arm64-6](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-arm64/55263/console), [test-equinix_mnx-smartos20-x64-4](https://ci.nodejs.org/job/node-test-commit-smartos/nodes=smartos20-64/57244/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62965/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/62987/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-arm64/55263/console">Example</a></summary>

```
not ok 4127 parallel/test-stream-readable-unpipe-resume
  ---
  duration_ms: 360091.29500
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>es-module/test-typescript</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/62976/), [https://github.com/nodejs/node/pull/55332/](https://ci.nodejs.org/job/node-test-pull-request/63047/)) |
| Appeared | [test-ibm-ubuntu2204_sharedlibs_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46905/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46847/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62976/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63047/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46905/console">Example</a></summary>

```
not ok 660 es-module/test-typescript
  ---
  duration_ms: 120083.87200
  severity: fail
  exitcode: -15
  stack: |-
    timeout
    ✔ execute a TypeScript file (531.594394ms)
    ✔ execute a TypeScript file with imports (509.983192ms)
    ✔ execute a TypeScript file with imports (358.374002ms)
    ✔ execute a TypeScript file with imports with default-type module (482.736152ms)
    ✔ execute a TypeScript file with node_modules (425.564324ms)
    ✔ execute a TypeScript file with node_modules with default-type module (421.980899ms)
    ✔ expect error when executing a TypeScript file with imports with no extensions (397.919101ms)
    ✔ expect error when executing a TypeScript file with imports with no extensions with default-type module (485.466143ms)
    ✔ expect error when executing a TypeScript file with enum (385.478131ms)
    ✔ expect error when executing a TypeScript file with experimental decorators (391.712494ms)
    ✔ expect error when executing a TypeScript file with namespaces (388.117777ms)
    ✔ exec...
```
</details>

-------

| Reason | <code>parallel/test-file-write-stream4</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/55339/](https://ci.nodejs.org/job/node-test-pull-request/63016/), [https://github.com/nodejs/node/pull/55312/](https://ci.nodejs.org/job/node-test-pull-request/63056/)) |
| Appeared | [test-equinix-ubuntu2204_container-armv7l-1](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-armv7l/55313/console), [test-equinix-ubuntu2204_container-armv7l-2](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-armv7l/55286/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/63016/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63056/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-armv7l/55313/console">Example</a></summary>

```
not ok 3925 parallel/test-file-write-stream4
  ---
  duration_ms: 360079.86500
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------


### Jenkins Failure

| Reason | <code>Build timed out (after 30 minutes). Marking the build as failed.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 9 ([https://github.com/nodejs/node/pull/55217/](https://ci.nodejs.org/job/node-test-pull-request/63015/), [https://github.com/nodejs/node/pull/55332/](https://ci.nodejs.org/job/node-test-pull-request/63019/), [https://github.com/nodejs/node/pull/55333/](https://ci.nodejs.org/job/node-test-pull-request/63020/), [https://github.com/nodejs/node/pull/55323/](https://ci.nodejs.org/job/node-test-pull-request/63021/), [https://github.com/nodejs/node/pull/55348/](https://ci.nodejs.org/job/node-test-pull-request/63024/), [https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/63025/), [https://github.com/nodejs/node/pull/55329/](https://ci.nodejs.org/job/node-test-pull-request/63029/), [https://github.com/nodejs/node/pull/55339/](https://ci.nodejs.org/job/node-test-pull-request/63030/), [https://github.com/nodejs/node/pull/55114/](https://ci.nodejs.org/job/node-test-pull-request/63031/)) |
| Appeared | [test-azure_msft-win11_vs2022-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2019/30772/console), [test-azure_msft-win11_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022/30778/console), [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2019-arm64/30772/console), [test-azure_msft-win11_vs2022-arm64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30778/console), [test-rackspace-win2016_vs2015-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2019-x86/30772/console), [test-rackspace-win2016_vs2015-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022-x86/30772/console), [test-rackspace-win2022_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2019/30772/console), [test-rackspace-win2022_vs2022-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/30778/console), [test-azure_msft-win2016_vs2017-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2016-COMPILED_BY-vs2019-x86/30772/console), [test-rackspace-win2019_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2019-COMPILED_BY-vs2019/30769/console), [test-rackspace-win2022_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2022-COMPILED_BY-vs2019/30772/console), [test-rackspace-win2022_vs2022-x64-6](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2022-COMPILED_BY-vs2022/30778/console), [test-azure_msft-win11_vs2022-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2019/30772/console), [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30778/console), [test-azure_msft-win11_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-COMPILED_BY-vs2019/30772/console), [test-azure_msft-win2016_vs2017-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2019-x86/30772/console), [test-rackspace-win2022_vs2022-x64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2022-COMPILED_BY-vs2019/30772/console), [test-rackspace-win2019_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2019-COMPILED_BY-vs2022/30777/console), [test-rackspace-win2019_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/30777/console), [test-azure_msft-win11_vs2022-arm64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30776/console), [test-rackspace-win2022_vs2022-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/30776/console), [test-azure_msft-win11_vs2022-arm64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30776/console), [test-azure_msft-win11_vs2022-arm64-6](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30776/console), [test-rackspace-win2019_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2019-COMPILED_BY-vs2022/30776/console), [test-azure_msft-win10_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-native-suites/nodes=win10-vs2019-COMPILED_BY-vs2019/25138/console), [test-azure_msft-win10_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-native-suites/nodes=win10-vs2019-COMPILED_BY-vs2022/25154/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-native-suites/nodes=win10-vs2019-COMPILED_BY-vs2022/25152/console), [test-azure_msft-win10_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2019/30749/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/63015/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63031/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2019/30772/console">Example</a></summary>

```
Build timed out (after 30 minutes). Marking the build as failed.
```
</details>

-------

| Reason | <code>Cannot run program "sh" (in directory "c:\workspace\node-test-binary-windows-js-suites"): CreateProcess error=2, The system cannot find the file specified</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/55348/](https://ci.nodejs.org/job/node-test-pull-request/63024/), [https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/63025/), [https://github.com/nodejs/node/pull/55329/](https://ci.nodejs.org/job/node-test-pull-request/63029/), [https://github.com/nodejs/node/pull/55339/](https://ci.nodejs.org/job/node-test-pull-request/63030/), [https://github.com/nodejs/node/pull/55114/](https://ci.nodejs.org/job/node-test-pull-request/63031/)) |
| Appeared | [test-azure_msft-win10_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2019/30772/console), [test-azure_msft-win10_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win10-COMPILED_BY-vs2019/30772/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2019/30772/console), [test-azure_msft-win10_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win10-COMPILED_BY-vs2022/30771/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/63024/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63031/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2019/30772/console">Example</a></summary>

```
java.io.IOException: Cannot run program "sh" (in directory "c:\workspace\node-test-binary-windows-js-suites"): CreateProcess error=2, The system cannot find the file specified
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at Jenkins v2.462.2//hudson.Proc$LocalProc.<init>(Proc.java:252)
	at Jenkins v2.462.2//hudson.Proc$LocalProc.<init>(Proc.java:221)
	at Jenkins v2.462.2//hudson.Launcher$LocalLauncher.launch(Launcher.java:994)
```
</details>

-------

| Reason | <code>Backing channel 'JNLP4-connect connection from ... is disconnected.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/55332/](https://ci.nodejs.org/job/node-test-pull-request/63006/), [https://github.com/nodejs/node/pull/55287/](https://ci.nodejs.org/job/node-test-pull-request/63007/), [https://github.com/nodejs/node/pull/55339/](https://ci.nodejs.org/job/node-test-pull-request/63030/), [https://github.com/nodejs/node/pull/55114/](https://ci.nodejs.org/job/node-test-pull-request/63031/)) |
| Appeared | [test-rackspace-win2019_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/30778/console), [test-azure_msft-win11_vs2022-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2022/30778/console), [test-rackspace-win2019_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/30778/console), [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30777/console), [test-azure_msft-win11_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-COMPILED_BY-vs2022/30777/console), [test-azure_msft-win11_vs2022-arm64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30777/console), [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/30777/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_icu_x64/46875/console), [test-osuosl-ubuntu2204_container-armv7l-1](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-armv7l/55281/console), [test-osuosl-ubuntu2204_container-arm64-2](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-arm64/55280/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/63006/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63031/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/30778/console">Example</a></summary>

```
java.io.IOException: Backing channel 'JNLP4-connect connection from 104.239.175.247/104.239.175.247:49713' is disconnected.
	at hudson.remoting.RemoteInvocationHandler.channelOrFail(RemoteInvocationHandler.java:215)
	at hudson.remoting.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:285)
	at jdk.proxy2/jdk.proxy2.$Proxy199.isAlive(Unknown Source)
	at hudson.Launcher$RemoteLauncher$ProcImpl.isAlive(Launcher.java:1212)
	at hudson.Launcher$RemoteLauncher$ProcImpl.join(Launcher.java:1204)
```
</details>

-------


### Git Failure


### Build Failure

| Reason | <code>Error: invalid syntax</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/55160/](https://ci.nodejs.org/job/node-test-pull-request/62962/), [https://github.com/nodejs/node/pull/53160/](https://ci.nodejs.org/job/node-test-pull-request/62975/), [https://github.com/nodejs/node/pull/55329/](https://ci.nodejs.org/job/node-test-pull-request/63004/)) |
| Appeared | [test-rackspace-debian10-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=debian10-x64/60964/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/62962/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63004/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=debian10-x64/60964/console">Example</a></summary>

```
Error: invalid syntax
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack at ChildProcess.<anonymous> (/home/iojs/build/workspace/node/deps/npm/node_modules/node-gyp/lib/configure.js:317:18)
gyp ERR! stack at ChildProcess.emit (node:events:519:28)
gyp ERR! stack at ChildProcess._handle.onexit (node:internal/child_process:294:12)
```
</details>

-------

| Reason | <code>ERROR: Step ?Publish JUnit test result report? failed: No test report files were found. Configuration error?</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/55239/](https://ci.nodejs.org/job/node-test-pull-request/63011/), [https://github.com/nodejs/node/pull/55337/](https://ci.nodejs.org/job/node-test-pull-request/63012/)) |
| Appeared | [test-equinix-ubuntu2204_container-arm64-5](https://ci.nodejs.org/job/node-test-commit-arm-debug/nodes=ubuntu2204_debug-arm64/15424/console), [test-equinix_mnx-smartos20-x64-3](https://ci.nodejs.org/job/node-test-commit-smartos/nodes=smartos20-64/57262/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/63011/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63012/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-arm-debug/nodes=ubuntu2204_debug-arm64/15424/console">Example</a></summary>

```
ERROR: Step ?Publish JUnit test result report? failed: No test report files were found. Configuration error?
Collecting metadata...
Metadata collection done.
Notifying upstream projects of job completion
Finished: FAILURE
```
</details>

-------

| Reason | <code>fatal: No rebase in progress?</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/55156/](https://ci.nodejs.org/job/node-test-pull-request/63038/), [https://github.com/nodejs/node/pull/55348/](https://ci.nodejs.org/job/node-test-pull-request/63039/)) |
| Appeared | [test-ibm-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46897/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-6](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_smallicu_x64/46896/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/63038/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/63039/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/46897/console">Example</a></summary>

```
fatal: No rebase in progress?
```
</details>

-------

